versions/blockspacerace_versions.js

```diff
@@ -1,9 +1,9 @@
 const blockspaceraceVersions = Object.freeze({
-  "app-latest-tag": "v0.13.0",
-  "app-latest-sha": "6ba60e18fec27d3e3a42a71be082e255913cf4cf",
-  "core-latest-tag": "v1.18.0-tm-v0.34.27",
-  "core-latest-sha": "9bfa856909861081d2bbc715ee00ed56e9fc43bb",
-  "node-latest-tag": "v0.9.2",
-  "node-latest-sha": "4d31ad4a537e41b5ac588e14cf807904abbb8d8a",
+  "app-latest-tag": "v0.13.2",
+  "app-latest-sha": "1c51eb36f5e733e7cd0eefef67d7da9202968b87",
+  "core-latest-tag": "v1.19.0-tm-v0.34.27",
+  "core-latest-sha": "2a26e7a5ffb5451bf12d09fe9a88a15c6f867b62",
```

Comment:
[non-blocking][informational] this is the latest celestia-core release but it's not the celestia-core release used in celestia-app 0.13.2. See https://github.com/celestiaorg/celestia-app/blob/1c51eb36f5e733e7cd0eefef67d7da9202968b87/go.mod#L189

Given this field isn't used in docs, it seems fine to update to latest

Reply:
ack, thank you